.github/workflows/ci.yml

```
@@ -0,0 +1,24 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Maximize Build Space
+        uses: ublue-os/remove-unwanted-software@cc0becac701cf642c8f0a6613bbdaf5dc36b259e # v9
+        with:
+          remove-codeql: true
+
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Build ISO
+        id: build-iso
+        uses: zeglius/titanoboa@compose-action
```

```suggestion
        uses: ublue-os/titanoboa@main
```
Lets wait for this Zeglius's PR to get merged then apply this suggestion